runtime/lua/vim/treesitter.lua

```diff
@@ -452,6 +452,9 @@ function M.show_tree(opts)
       pg:draw(b)
     end,
   })
+  a.nvim_buf_set_keymap(b, 'n', 'q', '<cmd>q!<cr>', {
+    desc = 'Close playground',
```

Comment:
A bit unsure on how to call this `playground` is not shown publicly. `Close tree-sitter syntax window`?

Reply:
Since the map is buffer-local, I would think it should simply be "Close the current window" (or "Quit the current window"—the docs seem to use both terms interchangeably).